src/libsyntax/parse/parser.rs

```diff
@@ -4284,7 +4284,7 @@ impl<'a> Parser<'a> {
                                    -> PResult<'a, Option<Stmt>> {
         maybe_whole!(self, NtStmt, |x| Some(x));
 
-        let attrs = self.parse_outer_attributes()?;
+        let attrs = self.parse_outer_attributes(true)?;
```

Comment:
why are we passing `true` here?

Reply:
I have to confirm, but IIRC it is because otherwise `'label: loop {}` would have been rejected (before I added an explicit check for a `:`).